capture/reader-libpcap-file.c

```diff
@@ -196,6 +202,51 @@ LOCAL int reader_libpcapfile_process(char *filename)
         return 1;
     }
 
+    // check to see if viewer might have access issues to non-copied pcap file
+    if (config.copyPcap == 0) {
+
+      if (strlen (filename) > 255) {
+        // filename bigger than path buffer, skip check
+      } else if ((config.dropUser == NULL) && (config.dropGroup == NULL)) {
+        // drop.User,Group not defined -- skip check
+      } else if (strncmp (filename, "/", 1) != 0) {
+        LOG("WARNING using a relative path may make pcap inaccessible to viewer");
+      } else {
+
+    		path[0] = 0;
+
+        p = strtok (filename, "/");
```

Comment:
need to use strtok_r

Reply:
thanks for comments.  making changes.  i also realized I had these two vars defined in the middle of the function-- do you want the definitions at the top at the function level?

          if (stat(path, &stats) != -1) {
            struct group *gr = getgrgid (stats.st_gid);
            struct passwd *pw = getpwuid (stats.st_uid);

Reply:
nope that is fine there, other than the gr/pw should be lined up. :)